tests/test_synthetic_volume.py

```diff
@@ -96,8 +100,15 @@ def test_volume_repr(vol_fixture):
 
 
 def test_volume_generate(vol_fixture):
-    """Test that a volume is generated"""
-    _ = vol_fixture.generate()
+    """
+    Test that a volume is generated
+    and stores pixel_size when provided.
+    """
+    v = vol_fixture.generate()
+
+    # In vol_fixture, we assign pixel_size to volumes having odd voxel sizes.
+    if vol_fixture.L % 2:
+        assert v.pixel_size == 3
```

Comment:
This will flake on some instances, no (or is 3.0 exactly representable in floating point?)?

Reply:
Great. Thanks!